app/src/main/java/com/kickstarter/viewmodels/BackingFragmentViewModel.kt

```diff
@@ -189,6 +189,7 @@ interface BackingFragmentViewModel {
                     .subscribe(this.backerAvatar)
 
             this.projectDataInput
+                    .filter { it.project().isBacking || ProjectUtils.userIsCreator(it.project(), it.user()) }
```

Comment:
Should we add a test for this or is it already covered by another test?

Reply:
Great point, I'll add a test

Reply:
Done! two new tests, 
- one checking as creator being able to see the reward of the backer.
- one for checking no rewad information if the user is neither a creator or a backer